tests/integration/docusaurus/connecting_to_your_data/how_to_introspect_and_partition_your_data/sql_database/yaml_example_complete.py

```diff
@@ -94,7 +99,7 @@
 
 # Please note this override is only to provide good UX for docs and tests.
 # In normal usage you'd set your data asset name directly in the BatchRequest above.
-batch_request.data_asset_name = "yellow_tripdata_sample_2019_01"
+batch_request.data_asset_name = "main.yellow_tripdata_sample_2019_01"
```

Comment:
I would type hint all of these since they are overrides that aren't exposed in the docs.

Reply:
5 total